commercial/app/model/merchandise/events/Eventbrite.scala

```diff
@@ -3,185 +3,108 @@ package model.commercial.events
 import java.lang.System._
 
 import commercial.feeds.{FeedMetaData, MissingFeedException, ParsedFeed, SwitchOffException}
+import model.commercial.readsSeq
 import common.{ExecutionContexts, Logging}
 import org.joda.time.DateTime
-import play.api.data.validation.ValidationError
 import play.api.libs.functional.syntax._
 import play.api.libs.json._
-
 import scala.concurrent.Future
 import scala.concurrent.duration._
 import scala.util.control.NonFatal
 
 object Eventbrite extends ExecutionContexts with Logging {
 
-  case class EBResponse(pagination: EBPagination, events: Seq[EBEvent])
-
-  object EBResponse {
+  case class Response(pagination: Pagination, events: Seq[Event])
+
+  case class Pagination(page_number: Int, page_count: Int)
```

Comment:
See https://github.com/guardian/frontend/pull/14654#discussion_r82986121

Happy to reinstantiate the camel case; having looked at the way that Membership parse Eventbrite models, they do it with snake-case as I've done here. We can use camel case, but it does make sense to make our case classes reflect as closely as possible the actual feed, since they are inextricably linked. Happy to get rid of that though if it isn't deemed useful enough.